bedwars-plugin/src/main/java/com/andrei1058/bedwars/shop/main/BuyItem.java

```diff
@@ -227,11 +227,12 @@ public void give(Player player, IArena arena) {
                 if (permanent) nms.setUnbreakable(im);
 
                 if (i.getType() == Material.BOW) {
-                    if (permanent) nms.setUnbreakable(im);
+                    nms.setUnbreakable(im);
                     for (TeamEnchant e : arena.getTeam(player).getBowsEnchantments()) {
                         im.addEnchant(e.getEnchantment(), e.getAmplifier(), true);
                     }
                 } else if (nms.isSword(i) || nms.isAxe(i)) {
+                    nms.setUnbreakable(im);
```

Comment:
line 227, if the current item is set as permanent it will be unbreakable, so it is a configuration matter